src/stories/SelectionList.stories.tsx

```diff
@@ -189,46 +189,49 @@ function WithAlternateText(args) {
         return {...section, data};
     });
 
-    const onSelectRow = (item) => {
-        _.forEach(sections, (section) => {
-            const newSelectedIndex = _.findIndex(section.data, (option) => option.keyForList === item.keyForList);
+    const onSelectRow = (item: ListItem) => {
+        sections.forEach((section) => {
+            const newSelectedIndex = section.data.findIndex((option) => option.keyForList === item.keyForList);
 
             if (newSelectedIndex >= 0) {
-                setSelectedIndex(newSelectedIndex + section.indexOffset);
+                setSelectedIndex(newSelectedIndex + (section?.indexOffset ?? 0));
             }
         });
     };
     return (
-        <SelectionList
+        <SelectionListWithNavigation
             // eslint-disable-next-line react/jsx-props-no-spreading
-            {...args}
+            {...props}
             sections={sections}
             onSelectRow={onSelectRow}
+            ListItem={RadioListItem}
         />
     );
 }
 
 WithAlternateText.args = {
-    ...Default.args,
+    sections: SECTIONS,
+    onSelectRow: () => {},
+    initiallyFocusedOptionKey: 'option-2',
```

Comment:
Why have you decided to update it?

Reply:
I was trying to fix something and forgot to remove it, reverted!